roq-frontmatter/deployment/src/main/java/io/quarkiverse/roq/frontmatter/deployment/scan/RoqFrontMatterScanProcessor.java

```diff
@@ -163,8 +166,13 @@ private static Consumer<Path> addBuildItem(Path root, HashSet<RoqFrontMatterRawT
                     final String content = stripFrontMatter(fullContent);
                     String dateString = null;
                     if (fm.containsKey(DATE_KEY)) {
-                        ZonedDateTime date = ZonedDateTime.parse(fm.getString(DATE_KEY),
-                                DateTimeFormatter.ofPattern(config.dateFormat()));
+                        ZonedDateTime date = new DateTimeFormatterBuilder().appendPattern("yyyy-MM-dd[ HH:mm:ss Z]")
+                                .parseDefaulting(ChronoField.HOUR_OF_DAY, 12)
+                                .parseDefaulting(ChronoField.MINUTE_OF_HOUR, 0)
+                                .parseDefaulting(ChronoField.SECOND_OF_MINUTE, 0)
+                                .toFormatter()
+                                .withZone(ZoneId.of("UTC"))
+                                .parse(fm.getString(DATE_KEY), ZonedDateTime::from);
```

Comment:
Here we should use the filename date as default value (if present)

Reply:
for example `2024-08-29-welcome-to-roq.md`

Reply:
That wasn't here originally right? That is something new you want to introduce?

Reply:
Yes